packages/web/src/components/ai-attribution-modal/DropdownInput.jsx

```diff
@@ -4,7 +4,7 @@ import Select from 'antd/lib/select'
 import cn from 'classnames'
 import PropTypes from 'prop-types'
 
-import { ReactComponent as IconCaretDown } from 'assets/img/iconCaretDown.svg'
+import IconCaretDown from 'assets/img/iconCaretDown.svg'
```

Comment:
For what it's worth this is the exact same change that we made in harmony for icons so it's good that we are an automatically consistent now

Reply:
Nice